runtime/near-test-contracts/src/lib.rs

```diff
@@ -28,14 +28,19 @@ pub fn sized_contract(size: usize) -> Vec<u8> {
 
 /// Standard test contract which can call various host functions.
 ///
-/// Note: the contract relies on the latest protocol version, and
-/// might not work for tests using an older version.
+/// Note: the contract relies on the latest protocol version, and might not work
+/// for tests using an older version. In particular, if a test depends on a
+/// specific protocol version, it should use [`base_rs_contract`].
```

Comment:
What’s `nightly_rs_contract` then?

Reply:
rs_contract is latest stable protocol, nightly is latest nightly protocol. Pushed a clarifying commit!

Reply:
This is still confusing to me. Is this accurate:
- `rs_contract` → protocol as defined in master branch,
- `base_rs_contract` → protocol as currently running in production,
- `nightly_rs_contract` → nightly protocol as defined in master branch?
The phrasing ‘for tests using an older version’ doesn’t really explain things.

Reply:
That's correct for `rs` and `nightly`. For `base`, that is literally the oldest possible protocol version (which I think is 29 for mainnet, judging from genesis.config). Tried to clarify more :)